### DISCUSSION TITLE
Can I Call Chi or Pon on the last tile?

### DISCUSSION THREAD

**Discussion by shinkuan** [smly/mjai.app#38] [General]
Just like title asked, can you call chi or pon on the last tile?
Kan is not possible on last tile because there is no tile left to zimo, but what about chi or pon?

**Comment by smly** [comment]
Thanks for the question. In general rules, the houteihai (河底牌; last discard) is fixed and is not allowed to be moved.
Not sure if you can move houteihai on mjai.app simulator. If it is possible, I will fix it.

**Reply by smly** [reply]
Confirmed that the simulator (libriichi) is validating chi or pon on the last discard.

https://github.com/smly/mjai.app/blob/b53fa40e645313b9eba861d2fbcf38bf7f32f19e/src/state/action.rs#L137-L147

**Reply by shinkuan** [reply]
Thanks!
